packages/renderer/src/lib/preferences/PreferencesConnectionCreationRendering.svelte

```diff
@@ -194,7 +194,13 @@ async function handleValidComponent() {
     data[key] = value;
   }
 
-  connectionAuditResult = await window.auditConnectionParameters(providerInfo.internalId, data as AuditRequestItems);
+  try {
+    connectionAuditResult = await window.auditConnectionParameters(providerInfo.internalId, data as AuditRequestItems);
+  } catch (err: unknown) {
+    if (err instanceof Error) {
```

Comment:
Why do we have this special case (compared to line 90) ?

Reply:
I don't get why we need a special case for Error and not in other parts of the codebase